docs/changelog.md

```diff
@@ -8,6 +8,8 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.1.0/).
 
 ### Added
 
+- Added `uploading` property to `six-file-upload`.
+
 ### Changed
 
 ### Fixed
```

Comment:
The part which was fixed for #204 is not mentioned in the changelog, please mention this here

Reply:
Point taken (sorry this was my very first PR...).
Just added a commit with the comment in the changelog

Reply:
No worries :) 
It's just for future PRs that you are aware of it - we definitely have some additional documentation to do on the contribution guide to make it clearer :) 

thanks for updating the changelog accordingly